pkg/scheduler/internal/cache/comparer/comparer.go

```diff
@@ -66,10 +84,12 @@ func (c *cacheComparer) Compare() error {
 	return nil
 }
 
-type compareStrategy struct {
+// CompareStrategy is an implementation of the Scheduler's cache compare strategy.
+type CompareStrategy struct {
 }
 
-func (c compareStrategy) CompareNodes(nodes []*v1.Node, nodeinfos map[string]*schedulercache.NodeInfo) (missed, redundant []string) {
+// CompareNodes compares actual nodes with cached nodes.
+func (c CompareStrategy) CompareNodes(nodes []*v1.Node, nodeinfos map[string]*schedulercache.NodeInfo) (missed, redundant []string) {
```

Comment:
If you do a follow-up PR. Maybe these methods can be moved onto the CacheComparer type and CompareStrategy can be deleted. I don't see the benefit of having it as a separate, embedded type.

Reply:
Understand